theano/gof/tests/test_op.py

```diff
@@ -233,6 +235,28 @@ def c_code(self, node, name, inputs, outputs, sub):
             self.assertRaises((NotImplementedError, utils.MethodNotDefined),
                               thunk)
 
+    def test_no_make_node(self):
+        class IncOne(Op):
+            """An Op without make_node"""
+            __props__ = ()
+            itypes = [T.fmatrix]
+            otypes = [T.fmatrix]
+
+            def perform(self, node, inputs, outputs):
+                input, = inputs
+                output, = outputs
+                output[0] = input + 1
+
+        x_input = T.fmatrix('x')
+        o = IncOne()(x_input)
+
+        # Confirming that make_node method is implemented
```

Comment:
@nouiz , since this functionality should be working, i've written the test this way, without implementing the "make_node" method and just defining the itypes and otypes.
So, I should instead write the test to check if the operation performed on the "perform" method has updated the value in "output" variable ?